lib/omnisharp-atom/features/code-action.ts

```diff
@@ -84,6 +85,25 @@ class CodeAction implements OmniSharp.IFeature {
         }));
     }
 
+    private getRequest(client: OmniSharp.ExtendApi, codeAction?: number) {
+        var editor = atom.workspace.getActiveTextEditor();
+        var range = editor.getSelectedBufferRange();
+
+        var request = client.makeDataRequest<OmniSharp.Models.CodeActionRequest>({
+            WantsTextChanges: true,
+            SelectionStartLine: range.start.row + 1,
```

Comment:
It didn't work without them. Think he just did the Line and Column not the selection stuff